test/hooks/blueprints.test.js

```diff
@@ -0,0 +1,49 @@
+/* jshint expr:true */
+'use strict';
+
+var expect = require('chai').expect,
+    Microservice = require('../../index'),
+    supertest = require('supertest');
+
+
+var cwd,
+    request,
```

Comment:
![Codacy](https://www.codacy.com/versioned/images/favicon.png) Issue found: ["request" is defined but never used](https://www.codacy.com/app/chris-ludden/restify-microservice/file/1745573794/issues/source?bid=3045492&fileBranchId=3056295#l10)